.circleci/config.yml

```diff
@@ -125,10 +149,13 @@ jobs:
           keys:
             - reaction-v6-node-modules-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
             - reaction-v6-node-modules-{{ .Branch }}
+      - run:
+          name: Authenticate with registry
+          command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
```

Comment:
This will overwrite the contents of `.npmrc`. You might want to do `> ~/.npmrc` to avoid losing the contents of the repo file, and/or do `>>` instead of `>` to append to the existing file contents.